.gitignore

```diff
@@ -1,7 +1,7 @@
 query.log
 Corefile
 *.swp
-^coredns$
+coredns
```

Comment:
Won't this match `vendor/github.com/coredns`?

Reply:
Ah, yea, Ill drop this change

Reply:
Maybe `./coredns` would be what you want?

Reply:
I think `/coredns` will work. Ill add the gitignore commit back.